frontend/src/lib/components/AuthorizedUrlList/authorizedUrlListLogic.ts

```diff
@@ -47,6 +47,18 @@ export function sanitizePossibleWildCardedURL(url: string): URL {
     return new URL(deWildCardedURL)
 }
 
+/**
+ * Checks if the URL has a wildcard (*) in the port position eg https://*.example.com or http://localhost:*
```

Comment:
I think https://*.example.com isn't a correct example since the wildcard is in the subdomain, not the port position